backend/dal/Migrations/v01.05.06/Up/PostUp/01-RoleClaims.sql

```diff
@@ -0,0 +1,54 @@
+PRINT 'Adding Role'
+
+--Adding new role for the SRES Financial Reporter
+INSERT INTO
+    dbo.[Roles] (
+        [Id],
+        [Name],
+        [Description],
+        [IsPublic],
+        [IsDisabled],
+        [SortOrder]
+    )
+VALUES
+    (
+        '5c6cea5b-9b7c-47e8-852c-693e90ed815e',
+        'SRES Financial Reporter',
+        'The SRES Financial Reporter can view, create, and delete non-final SPL reports.',
+        0,
+        0,
+        0
+    );
+
+PRINT 'Adding Claim'
+--Adding new claim for the SRES Financial Manager
+INSERT INTO
+    dbo.[Claims] ([Id], [Name], [Description], [IsDisabled])
+VALUES
+(
+    '81ded21c-ed32-4694-8f33-79ef17833f2b',
+    'reports-spl-admin',
```

Comment:
Claims have a `KeycloakRoleId` value that maps to keycloak roles.  You should probably set this value too.

Reply:
Ok, I think I'm confusing 81ded21c-ed32-4694-8f33-79ef17833f2b with the claim id, when it seems like it's is actually the KeycloakRoleId. How do I find the id for this particular claim, as all I can see is the id field in the url which looks like it's the KeycloakRoleId http://localhost:8080/auth/admin/master/console/#/realms/pims/roles/81ded21c-ed32-4694-8f33-79ef17833f2b?

Reply:
You can make up your own, or just keep the on you have.

Reply:
Is the Claims.KeycloakRoleId field supposed to map to the Roles.Id field? I looked at these values in my local database and they don't seem to map to each other, so I'm wondering what this field actually maps to?